msteams-platform/webhooks-and-connectors/how-to/add-incoming-webhook.md

```diff
@@ -1,68 +1,91 @@
 
 * Bots
 * Messaging extensions
 * Connectors
 
-## Incoming webhook key features
+## Key features of incoming webhook
 
-| Feature | Description |
+The following table provides the features and description of incoming webhook:
+
+| Features | Description |
 | ------- | ----------- |
-|Scoped Configuration|Incoming webhooks are scoped and configured at the channel level. For example, outgoing webhooks are scoped and configured at the team level.|
-|Secure resource definitions|Messages are formatted as JSON payloads. This declarative messaging structure prevents the injection of malicious code as there is no code execution on the client.|
-|Actionable messaging support|If you choose to send messages via cards, you must use the **actionable message card** format. Actionable message cards are supported in all Office 365 groups including Teams. Here are links to the [Legacy actionable message card reference](/outlook/actionable-messages/message-card-reference) and the [Message card playground](https://messagecardplayground.azurewebsites.net).|
-|Independent HTTPS messaging support| Cards are a great way to present information in a clear and consistent way. Any tool or framework that can send HTTPS POST requests can send messages to Teams via an incoming webhook.|
-|Markdown support|All text fields in actionable messaging cards support basic Markdown. **Don't use HTML markup in your cards**. HTML is ignored and treated as plain text.|
+|Actionable messaging support|Actionable message cards are supported in all Office 365 groups including Teams. If you send messages through cards, you must use the actionable message card format. For more information, see [legacy actionable message card reference](/outlook/actionable-messages/message-card-reference) and [message card playground](https://messagecardplayground.azurewebsites.net).|
+|Independent HTTPS messaging support|Cards provide information clearly and consistently. Any tool or framework that can send HTTPS POST requests, can send messages to Teams through an incoming webhook.|
+|Markdown support|All text fields in actionable messaging cards support basic Markdown. Do not use HTML markup in your cards. HTML is ignored and treated as plain text.|
+|Scoped configuration|Incoming webhook is scoped and configured at the channel level.|
+|Secure resource definitions|Messages are formatted as JSON payloads. This declarative messaging structure prevents the insertion of malicious code.|
+
+> [!NOTE]
+> Teams bots, messaging extensions, incoming webhook, and the Bot Framework support Adaptive Cards, an open cross card platform framework. Currently, [Teams connectors](../../webhooks-and-connectors/how-to/connectors-creating.md) do not support Adaptive Cards. However, it is possible to create a [flow](https://flow.microsoft.com/blog/microsoft-flow-in-microsoft-teams/) that posts Adaptive Cards to a Teams channel.
```

Comment:
Add this link for reference to Adaptive cards and Incoming webhooks:  https://docs.microsoft.com/en-us/microsoftteams/platform/task-modules-and-cards/what-are-cards#adaptive-cards-and-incoming-webhooks